File: smp2017-web-Vivek/CS50/pset1/mario2.c
#include<stdio.h>
#include<cs50.h>
void main()
{
	printf("Height :")
	int n=get_int();
	while(n<0 || n>23){
		printf("Height :");
		n=get_int();
	}
	int i,j;
	for(i=1;i<=n;i++){
		for(j=1;j<=(n-i);j++){
			printf(" ");
		}
		for(k=1;k<=i;k++){
			printf("#");
		}
		printf("#\n");
	}
}

File: smp2017-web-Vivek/CS50/pset6/mario2.py
import cs50
def main():
print("Height:",end"")
n=cs50.get_int()

while True:
	if n>=0 and n<=23:
		break
	else:
		print("Height:",end"")
		n=cs50.get_int()

for i in range n:
	for j in range(i,n):
		print(" ",end"")	
	for k in range(1,i):
		print("#",end"")
	print("#\n")
	
if __name__="__main__":
	main()